src/opnsense/service/conf/actions.d/actions_dhcpd.conf

```diff
@@ -1,9 +1,15 @@
 [list.leases]
-command:/usr/local/opnsense/scripts/dhcp/get_leases.py /inactive %s
-parameters:%s
+command:/usr/local/opnsense/scripts/dhcp/get_leases.py
+parameters:/inactive %s /duplicates %s
 type:script_output
 message:list dhcp leases %s
 
+[list.static]
+command:/usr/local/opnsense/scripts/dhcp/get_staticmap.php
```

Comment:
BTW, I'm quite sure `pluginctl -r static_mapping` is shorter. we can figure out a solution for only grabbing ipv4. the rest ist standard arguments.

Reply:
The current construction is fairly simple and explicit, If this has the potential of being shorter it would likely require some combination of named parameters and a single argument to https://github.com/opnsense/core/blob/master/src/etc/inc/plugins.inc.d/dhcpd.inc#L44,  not entirely sure how this would fit into `pluginctl`.

Reply:
Moving family arg of dhcpd_staticmap to first would be an option. Also family is pretty easy to distinguish between returned ipaddr vs. ipaddrv6

Reply:
@fichtner how about https://github.com/opnsense/core/pull/6643/commits/d38ab331d0f745e734358ccf01b200d8480046f3?

Reply:
Looks good, thanks!